mozfun/event_analysis/create_events_view/README.md

```diff
@@ -0,0 +1,181 @@
+events.
+
+```
+event_count_name INT64
+```
+
+### Examples
+The following creates a few fields:
+- `collection_flow` is a funnel for those that started creating
+    a collection within Fenix, and then finished, either by adding
+    those tabs to an existing collection or saving it as a new
+    collection.
+- `collection_flow_saved` represents users who started the collection
+    flow then saved it as a new collection.
+- `number_of_collections_created` is the number of collections created
+- `number_of_collections_deleted` is the number of collections deleted
+
+```
+CALL procedures.create_funnels_view(
+  'fenix_collection_funnels',
+  'moz-fx-data-shared-prod',
+  'org_mozilla_firefox',
+
+  -- Funnels
+  [
+    STRUCT(
+      "collection_flow" AS funnel_name,
+      [STRUCT(
+        "started_collection_creation" AS step_name,
+        [STRUCT('collections' AS category, 'tab_select_opened' AS event_name)] AS events),
+      STRUCT(
+        "completed_collection_creation" AS step_name,
+        [STRUCT('collections' AS category, 'saved' AS event_name),
+        STRUCT('collections' AS category, 'tabs_added' AS event_name)] AS events)
+    ] AS funnel),
+
+    STRUCT(
+      "collection_flow_saved" AS funnel_name,
+      [STRUCT(
+        "started_collection_creation" AS step_name,
+        [STRUCT('collections' AS category, 'tab_select_opened' AS event_name)] AS events),
+      STRUCT(
+        "saved_collection" AS step_name,
+        [STRUCT('collections' AS category, 'saved' AS event_name)] AS events)
+    ] AS funnel)
+  ],
+
+  -- Event Counts
+  [
+    STRUCT(
+      "number_of_collections_created" AS count_name,
+      [STRUCT('collections' AS category, 'saved' AS event_name)] AS events
+    ),
+    STRUCT(
+      "number_of_collections_deleted" AS count_name,
+      [STRUCT('collections' AS category, 'removed' AS event_name)] AS events
+    )
+  ]
+);
+```
+
+From there, we can query a few things. For example, the fraction 
+of users who completed each step of the collection flow over time:
+```
+SELECT
+    submission_date,
+    COUNTIF(collection_flow.started_collection_creation) / COUNT(*) AS started_collection_creation,
+    COUNTIF(collection_flow.completed_collection_creation) / COUNT(*) AS completed_collection_creation,
+FROM
+    `moz-fx-data-shared-prod`.analysis.fenix_collection_funnels
+WHERE
+    submission_date >= DATE_SUB(current_date, INTERVAL 28 DAY)
+```
+
+Or we can see the number of collections created and deleted:
+```
+SELECT
+    submission_date,
+    SUM(number_of_collections_created) AS number_of_collections_created,
+    SUM(number_of_collections_deleted) AS number_of_collections_deleted,
+FROM
+    `moz-fx-data-shared-prod`.analysis.fenix_collection_funnels
+WHERE
+    submission_date >= DATE_SUB(current_date, INTERVAL 28 DAY)
+```
+
+#### Recommended Pattern
+Because the view definitions themselves are not informative about the contents of the events fields,
```

Comment:
This is probably worth putting higher up, it's a high-level concept that's worth communicating early.